Automattic/wp-calypso
# P2: Redirect to hub on user management pages

#### Changes proposed in this Pull Request

For P2 sites:

* On all pages related to user management, redirect to the P2 site's hub so user management is done only there.
* Any page starting with `/people` should redirect to the hub.

See `2888-gh-Automattic/p2`.

I'm not sure if there is an easier way to get the url to redirect to than using string replace? Suggestions very welcome. :)

#### Testing instructions

* On a non-hub, click "Users",  "All users", or "Add new" and verify that you are taken to the __hub__ – not the site you navigated from.
* On a non-hub enter url that is a `/people` page and verify that you are redirected. For instance: http://calypso.localhost:3000/people/email-followers/[site-you-are-testing-on].
